### DISCUSSION TITLE
Fluent For Steam 0.4alpha

### DISCUSSION THREAD

**Discussion by purogamer** [purogamer/Fluent-for-Steam#61] [Announcements]
<hr /><em>This discussion was created from the release <a href='https://github.com/purogamer/Fluent-for-Steam/releases/tag/0.4'>Fluent For Steam 0.4alpha</a>.</em>

**Comment by Benisgo** [comment]
The discord button in the skin does nothing

**Reply by Lilithbunny** [reply]
Lol

**Comment by OpenRift412** [comment]
It seems the three dots and the heart button in the compact variant have their menus swapped, because the heart shows steam-related menu items while the three dots links to your Patreon and such. Pretty sure it's supposed to be the other way around.
